area_detector_handlers/tests/conftest.py

```diff
@@ -53,6 +57,35 @@ def finalize():
     return (out_name, {}), (N_points, N_chans, N_bin, N_roi)
 
 
+@pytest.fixture(scope="module")
+def eigerfile(request):
+    N_chans = 4
+    N_points = 27
+    N_bin = 4096
+    N_roi = 32
+
+    seq_id = np.random.randint(10)
+    pre_fix = gettempprefix()
+    out_name = f'{pre_fix}_{seq_id}_master.h5'
+
+    with h5py.File("sample_data_000001.h5", "w") as dfile:
+        data = np.random.rand(N_points, N_chans, N_bin) * 1000
+        dfile["data_000001"] = data
+
+    with h5py.File(out_name, "w") as fout:
+        for e in EigerHandler.EIGER_MD_LAYOUT.values():
+            fout[e] = np.array([np.random.randint(1, 10)])
```

Comment:
Do we need `np.array()` here? I guess `np.random.randint()` should already return an ndarray.